DOCS-9646 add files to be edited. (#342)
mulesoft/docs-runtime-fabric: modules/ROOT/edits/mule-core-usage-self.adoc

> = Monitor Mule Core Usage With Runtime Fabric on Self-Managed Kubernetes

Reviewer: ```suggestion
= Monitor Mule Core Usage by Using Runtime Fabric on Self-Managed Kubernetes
```

> Anypoint Runtime Fabric on Self-Managed Kubernetes enables you to use your own infrastructure monitoring solution to view metrics, monitor and alert on conditions.

Reviewer: ```suggestion
Anypoint Runtime Fabric on Self-Managed Kubernetes enables you to use your own infrastructure monitoring solution to view metrics, monitor operations, and send alerts.
```

> Your monitoring solution can also be used to count and generate alerts on Mule core usage.

Reviewer: passive
```suggestion
You can also use your monitoring solution to count and generate Mule core usage alerts.
```

> 
> == Counting Mule Application Resources
> 
> The following procedures describe how to use your monitoring solution to aggregate resource usage across all Mule application containers on Runtime Fabric
> with Self-Managed Kubernetes. 

Reviewer: This does not need to be a separate subtopic:
```suggestion
To use your monitoring solution to aggregate resource usage across all Mule application containers:
```

> . Select all Pods containing a label equal to "MuleApplication"

Reviewer: Why init. cap. "Pods"?

Reviewer: What kind of label is this? If UI, bold; if syntax, mono.

Reviewer: End with period.

> . From each pod in the prior step, select the Container Name equal to "/app"

Reviewer: ```suggestion
. From each pod, select the Container Name equal to "/app".
```
ditto re. type styles and caps.

> . Use your monitoring solution to aggregate and plot the CPU Usage and CPU Request across each selected container.

Reviewer: Either lowercase CPU usage and request or format to clarify whether they are GUI labels or CLI syntax.

> Refer to your monitoring solution's documentation for information on how to aggregate, plot and alert on the CPU usage across the Mule applications as required.

Reviewer: information about; GLOBAL Oxford comma:
```suggestion
Refer to your monitoring solution documentation for information about how to aggregate, plot, and alert about CPU usage across the Mule applications.
```
